CHANGELOG.rst

```diff
@@ -2,17 +2,55 @@
 Change history
 ==============
 
-2.x.x
+2.7.0
 =====
-*March, 31, 2025*
+*April, 1, 2025*
+
+
+**New features**
+
+* [:open-klant:`212`] Expand BetrokkeneSerializer with digitale_adressen
+* [:open-klant:`239`] Accept Partij Identificatoren as part of Partij creation
+* [:open-klant:`355`] Add Notifications for InterneTaak and Partij
```

Comment:
```suggestion
* [:open-klant:`355`] Add Notifications for InterneTaak and Partij (see :ref:`installation_configuration_notificaties_api`)
```

Could you also add a line to state that extra configuration steps (one to configure external services and to configure notificaties) were added to configure notifications (and refer to `installation_configuration_cli`)?